apps/proxy-auth/src/app/otp/send-otp/send-otp.component.ts

```diff
@@ -63,7 +63,7 @@ export class SendOtpComponent extends BaseComponent implements OnInit, OnDestroy
     public otpWidgetData;
     public showRegistration = new BehaviorSubject<boolean>(false);
     public referenceElement: HTMLElement = null;
-    public showLogin: boolean = false;
+    public showLogin$: BehaviorSubject<boolean> = this.otpWidgetService.showlogin;
```

Comment:
we don't use $ in Subject variable 
```suggestion
    public showLogin: BehaviorSubject<boolean> = this.otpWidgetService.showlogin;
```